polyglot/piranha/src/models/language.rs

```diff
@@ -75,27 +75,31 @@ impl PiranhaLanguage {
 
 impl Default for PiranhaLanguage {
   fn default() -> Self {
-    let rules: Rules = parse_toml(include_str!("../cleanup_rules/java/rules.toml"));
-    let edges: Edges = parse_toml(include_str!("../cleanup_rules/java/edges.toml"));
-    PiranhaLanguage {
-      name: default_language(),
-      supported_language: SupportedLanguage::default(),
-      language: tree_sitter_java::language(),
-      rules: Some(rules),
-      edges: Some(edges),
-      scopes: parse_toml::<ScopeConfig>(include_str!("../cleanup_rules/java/scope_config.toml"))
-        .scopes()
-        .to_vec(),
-      comment_nodes: vec!["line_comment".to_string(), "block_comment".to_string()],
-    }
+    PiranhaLanguage::from(default_language().as_str())
   }
 }
 
 impl From<&str> for PiranhaLanguage {
   fn from(language: &str) -> Self {
     match language {
-      "java" => PiranhaLanguage::default(),
-      "go" => PiranhaLanguage {
+      JAVA => {
+        let rules: Rules = parse_toml(include_str!("../cleanup_rules/java/rules.toml"));
+        let edges: Edges = parse_toml(include_str!("../cleanup_rules/java/edges.toml"));
+        Self {
+          name: language.to_string(),
+          supported_language: SupportedLanguage::default(),
```

Comment:
Shouldn't this be `SupportedLanguage::Java` here? Since, again, we want to concentrate the "Java is the default" logic in a single place that can be changed later?